content/docs/next-release/operator.md

```diff
@@ -0,0 +1,668 @@
+This example defines a default sampling strategy that is probabilistic, with a 50% chance of the trace instances being sampled.
+
+Refer to the Jaeger documentation on [Collector Sampling Configuration](https://www.jaegertracing.io/docs/latest/sampling/#collector-sampling-configuration) to see how service and endpoint sampling can be configured. The JSON representation described in that documentation can be used in the operator by converting to YAML.
+
+## Finer grained configuration
+
+The custom resource can be used to define finer grained Kubernetes configuration applied to all Jaeger components or at the individual component level.
+
+When a common definition (for all Jaeger components) is required, it is defined under the `spec` node. When the definition relates to an individual component, it is placed under the `spec/<component>` node.
+
+The types of supported configuration  include:
+
+* [affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity) to determine which nodes a pod can be allocated to
+
+* [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+
+* [resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container) to limit cpu and memory
+
+* [tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) in conjunction with `taints` to enable pods to avoid being repelled from a node
+
+* [volumes](https://kubernetes.io/docs/concepts/storage/volumes/) and volume mounts
+
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: simple-prod
+spec:
+  strategy: production
+  storage:
+    type: elasticsearch
+    options:
+      es:
+        server-urls: http://elasticsearch:9200
+  annotations:
+    key1: value1
+  resources:
+    requests:
+      memory: "64Mi"
+      cpu: "250m"
+    limits:
+      memory: "128Mi"
+      cpu: "500m"
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: kubernetes.io/e2e-az-name
+            operator: In
+            values:
+            - e2e-az1
+            - e2e-az2
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 1
+        preference:
+          matchExpressions:
+          - key: another-node-label-key
+            operator: In
+            values:
+            - another-node-label-value
+  tolerations:
+    - key: "key1"
+      operator: "Equal"
+      value: "value1"
+      effect: "NoSchedule"
+    - key: "key1"
+      operator: "Equal"
+      value: "value1"
+      effect: "NoExecute"
+  volumeMounts:
+    - name: config-vol
+      mountPath: /etc/config
+  volumes:
+    - name: config-vol
+      configMap:
+        name: log-config
+        items:
+          - key: log_level
+            path: log_level
+```
+
+# Accessing the Jaeger Console (UI)
+<!-- TODO Add tabs shortcode -->
+
+{{< danger >}}
```

Comment:
@jpkrohling Is this necessary? as k8s and ocp are dealt with in different sections?

Reply:
Let's remove it. If we think users are confused and expecting an ingress when a CR is installed on OpenShift, we can add it back.